### ISSUE TITLE
the result 

### ISSUE THREAD

**Issue by CdNut** [CIA-Oceanix/TrAISformer#1]
Professor Duong Nguyen,
        Thank you very much for sharing the code, but using the default config I can't reproduce the results in the paper, I can just get (0.8647, 1.6386, 2.7692) for (1 hour, 2 hours, 3 hours), and the train loss will reduce to negative number, I can not understand the reason. Could you please give me some advice?

**Comment by duonginspace** [comment]
Hello,

Please note that the values given by the code are in km, while the values presented in the paper were converted to nautical mile. It seems to me that you got the correct results.
We used cross-entropy loss, it's normal that the training loss is negative.
